includes/class.imageuploader.php

```diff
@@ -95,22 +102,28 @@ public function save() {
         // check if file with same name exists in upload path
         if (is_file($image_path)) {
             $this->url = $image_url;
+            // If the image already exists, get the id to link it to the post
```

Comment:
When an image exists ( used before in another post or side-loaded when the first image in content is used as featured ) we need to find it and get the id

Reply:
There might be another way to get the id in this case, i am still looking...

Reply:
@gregory-claeyssens  I could not find a better way to do this
its kind of an edge case when an image is already used before and now used as a featured image,
the only other option is: do what was commented out a bit below ( if image path is used, rename and upload a new one ) 
what do you think